src/material-moment-adapter/adapter/index.ts

```diff
@@ -29,10 +29,6 @@ export * from './moment-date-formats';
       provide: DateAdapter,
       useClass: MomentDateAdapter,
       deps: [MAT_DATE_LOCALE, MAT_MOMENT_DATE_ADAPTER_OPTIONS]
```

Comment:
I think I found why the tests did not work. Now with the `MAT_MOMENT_DATE_ADAPTER_OPTIONS` in the `deps` all tests pass.